pkgs/dart_services/lib/src/analysis_server.dart

```diff
@@ -188,14 +242,15 @@ abstract class AnalysisServerWrapper {
             edit.offset, edit.offset + edit.length, edit.replacement);
       }
 
+      // TODO: Allow offset result to be null.
```

Comment:
Doesn't this code handle null?

Reply:
The protobuf definition doesn't; we pass `0` to represent no value. I can fix this in the newer protocol however; I'll remove this comment.